src/Serializer/SymfonySerializerAdapter.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the CsaGuzzleBundle package
+ *
+ * (c) Charles Sarrazin <charles@sarraz.in>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code
+ */
+
+namespace Csa\Bundle\GuzzleBundle\Serializer;
+
+use Symfony\Component\Serializer\Serializer as SymfonySerializer;
+
+class SymfonySerializerAdapter implements SerializerAdapterInterface
+{
+    /**
+     * @var SymfonySerializer
+     */
+    protected $serializer;
+
+    /**
+     * @param SymfonySerializer $serializer
+     */
+    public function __construct(SymfonySerializer $serializer)
+    {
+        $this->$serializer = $serializer;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function serialize($data, $format, $context = [])
+    {
+        if (!is_array($context)) {
```

Comment:
you can just type hint the context here ;) 
`public function serialize($data, $format,array $context = [])`